extensions/amp-kaltura-player/0.1/amp-kaltura-player.js

```diff
@@ -65,6 +67,9 @@ class AmpKaltura extends AMP.BaseElement {
     setIsMediaComponent(this.element);
 
     this.entryId_ = this.element.getAttribute('data-entryid') || 'default';
+
+    this.serviceUrl_ =
```

Comment:
No restrictions at all on the what the origin could be here?

Reply:
Every kaltura CE installation are possible, https://github.com/kaltura/platform-install-packages/releases/tag/Propus-16.5.0 could be done by everyone, it is like amp-iframe.
Do you have security concern's? Another solution would be to have a array with possible publishers and check if it is part of it. But this implied every publisher have to make a pull request and validation of it's setup.

Reply:
Got it.